### PR TITLE
Added option to set post excerpt length

### PR DESCRIPTION
**Pull request by adiraomj**
Implemented enhancement #159

### CODE REVIEW

**Comment by Misplon** [comment]
@adiraomj do you want to strengthen this by using:

`return absint( siteorigin_setting( 'blog_excerpt_length' ) );`

https://codex.wordpress.org/Function_Reference/absint

**Comment by adiraomj** [comment]
I don't think it will be necessary. What do you think @gregpriday?

File: inc/extras.php [hunk 1]
```diff
@@ -149,3 +153,13 @@ function vantage_footer_widget_style(){
...
+
+if ( ! function_exists( 'vantage_excerpt_length' ) ) :
+/*
+ * Filter the except length
+ */
+function vantage_excerpt_length( $length ) {
+	return siteorigin_setting( 'blog_excerpt_length' );
+}
+endif;
+add_filter( 'excerpt_length', 'vantage_excerpt_length', 999 );
```
Comment (gregpriday) [inline inc/extras.php]: What's the reason for the `999` priority? Ideally, we'd stick to 10, or if we really need to take preference then go with something like 15. No need to go much higher than that - it starts losing its meaning.

**Comment by gregpriday** [comment]
Escaping like that is only really necessary when the value is going to be displayed on the frontend, or it can be manipulated in some way. For example MySQL injection. This value is used as an argument in some safe functions.

http://php.net/manual/en/function.array-slice.php
http://php.net/manual/en/function.preg-split.php

This functions will do their own type-casting to integer.

That said, I wouldn;t mind using absint as a value sanitization argument (if it's not already).

File: inc/extras.php (hunk 1 above)
Comment (adiraomj) [inline inc/extras.php]: @gregpriday I used the priority from the documentation. 

https://codex.wordpress.org/Plugin_API/Filter_Reference/excerpt_length

In fact they have a note about the priority.

**Comment by Misplon** [comment]
@adiraomj Bro, when you get a chance to look at this, please, just drop a comment and I'll jump in. Thanks :)

**Comment by adiraomj** [comment]
@Misplon I have made all suggested changes. Please have a look :)

**Pull request merged**